charts/redpanda/values.yaml

```
@@ -584,6 +584,15 @@ statefulset:
       resources: {}
       securityContext: {}
       extraVolumeMounts: |-
+    controllers:
+      image:
+        tag: v23.2.5
+        repository: docker.redpanda.com/redpandadata/redpanda-operator
+      enabled: false # experimental, must opt in
+      resources: {}
+      securityContext: {}
+      run:
```

I wouldn't name it `run`, but rather `controllerNames`.
```suggestion
      controllerNames:
```